app/helpers/compliance_summary_helper.rb

```diff
@@ -10,7 +10,7 @@ def textual_compliance_status
     else
       compliant = @record.last_compliance_status
       date      = @record.last_compliance_timestamp
-      h[:image] = "100/#{compliant ? "check" : "x"}.png"
+      h[:icon] = "#{compliant ? "pficon pficon-ok" : "pficon pficon-error-circle-o"}"
```

Comment:
May want to drop the outer quotes .. `h[:icon] = compliant ? "pficon pficon-ok" : "pficon pficon-error-circle-o"`